src/ui_components/search_input.js

```diff
@@ -34,6 +34,8 @@ export default class SearchInput {
       const isMobile = isMobileDevice();
       const isActive = document.activeElement.id === inputElement.id;
       inputElement.value = '';
+      const topBarHandle = document.querySelector('.top_bar');
+      topBarHandle.classList.remove('top_bar--search_filled');
```

Comment:
Why is this needed? In any cas, looks like it's linked to my other comment and should be put in `initTopBar()` too.

Reply:
The "_filled" class is needed because sole special style will apply only when we have started writing in the field.

The DOM manipulation will be removed according to your comment